package-lock.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
Please remove this generated file from the PR, adding it to `.gitignore` would be a good option.

Reply:
Check out https://medium.com/@Quigley_Ja/everything-you-wanted-to-know-about-package-lock-json-b81911aa8ab8#1c9a
Require to commit `package-lock.json`

Reply:
Since this file hasn't been added to the repository yet, it should be segregated from the current issue which is trying to add Image/Video support. Let's create a separate issue for the `package-lock.json`.

Reply:
Done :+1: